test/unit_convex_hull.jl

```diff
@@ -55,6 +51,47 @@ for N in [Float64, Rational{Int}]
     @test ispermutation(convex_hull!([N[0, 1], N[0, 2], N[0, 3]]), [N[0, 1], N[0, 3]]) # three points on a vertical line
     @test convex_hull!([N[0, 1], N[0, 1], N[0, 1]]) == [N[0, 1]] # three equal points
 
+    # four vertex case in 2 dimentions
+    A = N[1, 0]
+    B = N[1, 1]
+    C = N[-1, 1]
+    D = N[-1, 0]
+    expr = [A, B, C, D]
+    points = [A, B, C, D]
+    @test  iscounterclockwise(convex_hull!(points), expr) # ABCD
+    points = [A, D, C, B]
+    @test  iscounterclockwise(convex_hull!(points), expr) # ADCB
+    points = [A, B, D, C]
+    @test  iscounterclockwise(convex_hull!(points), expr) # ABDC
+    points = [A, D, B, C]
+    @test  iscounterclockwise(convex_hull!(points), expr) # ADBC
+    points = [D, B, C, A]
+    @test  iscounterclockwise(convex_hull!(points), expr) # DBCA
```

Comment:
This test is redundant (covered by ADBC) and instead it should test the case DACB (which corresponds to the bug we saw).